### PR TITLE
added product method functionality

### PR DESCRIPTION
**Pull request by chromaticity**
sorry, first time contributing to a repository like this :) I hope you don't mind but I've also added a new test case just to make sure that multiplying floats works. Let me know if there are any problems.

### CODE REVIEW

**Comment by BlakeGuilloud** [comment]
Thanks for the contribution @chromaticity ! If you could please create a new skeleton method with a test case for someone else to work on, I will merge this in! Thanks!

**Comment by chromaticity** [comment]
My bad! I'll do that right now

**Comment by chromaticity** [comment]
added in skeleton code along with my test cases... and it appears someone and I have thought of the same skeleton. lol

File: test/product.test.js [hunk 1]
```diff
@@ -6,4 +6,8 @@ test('multiplies 1 * 2 to equal 2', () => {
 
 test('multiplies 3 * 4 to equal 12', () => {
   expect(product(3, 4)).toBe(12);
+});
+
+test('multiplies 1.5 * 1.5 to equal 12', () => {
```
Comment (BlakeGuilloud) [inline test/product.test.js]: Everything looks great! If you don't mind fixing this test case "to equal 2.25" I'll merge away. Sometimes C & P gets the best of us.

**Comment by BlakeGuilloud** [comment]
Thanks @chromaticity !

**Comment by chromaticity** [comment]
awesome! no problem :)

**Pull request merged**